src/main/java/com/example/bookshop/dto/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,27 @@
+package com.example.bookshop.dto;
+
+import com.example.bookshop.annotation.FieldMatch;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.Length;
+
+@Getter
+@Setter
+@FieldMatch({"password", "repeatPassword"})
```

Comment:
No reason to define values, you don`t use them. Please implement annotation to make it work with any 2 fields